src/test/java/com/transferzero/sdk/model/PayoutMethodDetailsTest.java

```diff
@@ -53,7 +54,6 @@
 import com.transferzero.sdk.model.PayoutMethodMobileProviderEnum;
 import com.transferzero.sdk.model.PayoutMethodNatureOfBusinessEnum;
 import com.transferzero.sdk.model.PayoutMethodPixKeyTypeEnum;
-import com.transferzero.sdk.model.PayoutMethodTransferReasonEnum;
```

Comment:
Same issue as on Java8 :( https://github.com/transferzero/transferzero-sdk-java8/pull/48#discussion_r1145186205

Reply:
Let me revert this line...